src/vector/index.js

```diff
@@ -365,8 +366,17 @@ async function loadApp() {
         }).catch(err => {
             console.error(err);
 
-            const errorMessage = err.translatedMessage
+            let errorMessage = err.translatedMessage
                 || _t("Unexpected error preparing the app. See console for details.");
+            errorMessage = <span>
+                {_t(
+                    "Your configuration appears to be invalid. Please correct the error below " +
```

Comment:
Hmm, if anything, this makes it sound more like it's the user's fault than before?

Only people that have no session already will see this now, so it's less important than before, but how about "This installation of Riot seems to have an invalid server configuration. If you are the administrator, please correct the error below" or something?